### PR TITLE
adds a admin only sharp toolbox

### PR DESCRIPTION
**Pull request by boodaliboo**
### Intent of your Pull Request
Capture

#### Changelog

:cl:  
rscadd: Added new toolbox  
/:cl:

### CODE REVIEW

**Comment by boodaliboo** [comment]
IT COMPILES YES

File: code/game/objects/items/storage/toolbox.dm [hunk 1]
```diff
@@ -301,3 +301,14 @@
 	else
 		to_chat(user, "<span class='warning'>You need 10 floor tiles to start building a floorbot!</span>")
 		return
+
+
+/obj/item/storage/toolbox/yogurt
```
Comment (nesisknesis) [inline code/game/objects/items/storage/toolbox.dm]:
```suggestion
/obj/item/storage/toolbox/admin
```

Comment (nesisknesis) [reply code/game/objects/items/storage/toolbox.dm]: be informative

**Comment by alexkar598** [comment]
This may sound rude... but

Why

**Comment by boodaliboo** [comment]
> This may sound rude... but
> 
> Why

i dont know

File: code/game/objects/items/storage/toolbox.dm [hunk 2]
```diff
@@ -301,3 +301,14 @@
...
+
+
+/obj/item/storage/toolbox/admin
+	name = "a toolbox from the heavens"
+	force = 20
+	throwforce = 40
+	throw_speed = 2
+	throw_range = 5
+	materials = list(MAT_METAL=10000, MAT_GLASS=6000)
+	toolspeed = 0.5
```
Comment (Fikou) [inline code/game/objects/items/storage/toolbox.dm:313]: toolboxes arent any tool so this is useslsesls

Comment (SomeguyManperson) [reply code/game/objects/items/storage/toolbox.dm:313]: You fool if it's sharp it can be used to cut stuff and as a scalpel replacement in surgery

**Comment by alexkar598** [comment]
This is something any admin that can spawn items in the first place can do easily and if they dont know how, they maybe should try to learn basic varediting before bussing

**Pull request closed without merging**